filestack/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '3.2.1'
+__version__ = '3.2.2'
```

Comment:
this should be only changed when making releases, but don't worry, I'll take care of it in a moment.